_posts/2025-01-14-release-notes-3.3.5.md

```diff
@@ -0,0 +1,18 @@
+---
+category: announcement
+permalink: /news/3.3.5
+title: "Scala 3.3.5 is now available!"
+---
+Scala 3.3.5 LTS is now available!
+
+This patch release backports most of the improvements and bugfixes introduced in the Scala Next series up to the Scala 3.5.2 release.
+All of the backported changes were proven to not break either binary or source compatibility, by testing the compilation of over 1500 projects using [Scala 3 Open Community Build](https://github.com/VirtusLab/community-build3).
+
+Notable changes:
+
+- Scala 2 Standard Library upgraded to 2.13.15
+- Support for JDK 23
+- Added the `-Wall` option that enables all linting warnings
+- Aligned `-Wconf` parsing order with Scala 2 semantics
```

Comment:
This one could use a link to more information IMO, as it's especially like to have compatibility/migration implications, and since those implications might be silent (false negatives).

Reply:
https://github.com/scala/scala/releases/edit/v2.13.15 is a sample of how this might be handled (a prominent notice, with a link to further details)